web/src/pm_list.js

```diff
@@ -59,6 +61,19 @@ export function _build_private_messages_list() {
     return dom_ast;
 }
 
+export function clear_pm_search(e) {
+    e.stopPropagation();
+    const $input = $("#filter-pm-input");
+    if ($input.length) {
+        $input.val("");
+        $input.trigger("blur");
```

Comment:
I would have preferred if the focused stayed on the input after clicking on the clear button since I am not trying to close the expanded pm list by clicking on the close button but most likely trying to search for another user.

Reply:
Hmm, looks like we're not maintaining focus on any of the filter or search bar when we clear it by clicking on the cross right now. Is it because if someone wants to continue the search they may not want to touch the mouse and click on the icon and rather use `cmd + a`to select everything and type the new query? It still feels like it would be good to have it focussed anyway.

Reply:
Updated!

Reply:
> Hmm, looks like we're not maintaining focus on any of the filter or search bar when we clear it by clicking on the cross right now. 

Oh! You can add a commit to fix that in this PR too.

Reply:
or, you can start a frontend discussion for it and open a new PR based on the discussion.